### PR TITLE
KUVA-72 | Dockerized development environment

### PR DESCRIPTION
**Pull request by midori0507**
- Add docker config
- Add instruction on README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,10 +37,48 @@ To only start the client:
...
 ```
+
+## Useful docker command
+- To rebuild the docker images:
+    ```
+    $ docker-compose up --force-recreate --build
+    ```
+- To enter inside docker container environment:
+    ```
+    $ docker-compose exec web sh
```
Comment (igordavydsson) [inline README.md]: ` docker exec berth-frontend sh`?

Comment (midori0507) [reply README.md]: Should i name the image @igordavydsson ?

Comment (midori0507) [reply README.md]: @igordavydsson : Btw the script is correct, exec [SERVICE_NAME] not [CONTAINER_NAME]

Comment (igordavydsson) [reply README.md]: ah, I was thinking about `docker exec` not `docker-compose exec` 😄 my bad.

**Pull request merged**